website/en/docs/frameworks/redux.md

```diff
@@ -124,6 +124,44 @@ function bar(value: boolean): BarAction {
 }
 ```
 
+##### Typing Redux thunk actions <a class="toc" id="toc-typing-redux-thunk-actions" href="#toc-typing-redux-thunk-actions"></a>
+
+In order to type your Redux [thunk actions](http://redux.js.org/docs/advanced/AsyncActions.html#async-action-creators),
+you'll add types for `ThunkAction` as a function `Dispatch`, and `GetState`. `GetState` is a function that returns an `Object`. `Dispatch` accepts a disjoint union of `Action`, `ThunkAction`, `PromiseAction` and `Array<Action>` and can return `any`.
+
+```js
+type Dispatch = (action: Action | ThunkAction | PromiseAction) => any;
+type GetState = () => Object;
+type ThunkAction = (dispatch: Dispatch, getState: GetState) => any;
```

Comment:
according to the [source code](https://github.com/gaearon/redux-thunk/blob/master/src/index.js) `ThunkAction` can take an `extraArgument` if it was configured with it